plotly/tests/test_optional/test_figure_factory.py

```diff
@@ -11,6 +11,7 @@
 
 import numpy as np
 from scipy.spatial import Delaunay
+import os
```

Comment:
did `os` need to be imported for the added choropleth test? It doesn't look like it below.